src/ui/settingsdialog.cpp

```diff
@@ -396,7 +396,8 @@ void SettingsDialog::processDocsetList(const QJsonArray &list)
 {
     for (const QJsonValue &v : list) {
         QJsonObject docsetJson = v.toObject();
-        docsetJson[QStringLiteral("source")] = QStringLiteral("kapeli");
```

Comment:
Out of curiosity, why this one does not work with MSVC? There were two separate `QStringLiteral`s.